File: src/app/gkill/api/req_res/add_notification_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type AddNotificationResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	AddedNotification *reps.Notification `json:"added_notification"`
}

File: src/client/classes/api/req_res/add-notification-response.ts
'use strict'

import { Notification } from '@/classes/datas/notification'
import { GkillAPIResponse } from '../gkill-api-response'

export class AddNotificationResponse extends GkillAPIResponse {

    added_notification: Notification

    constructor() {
        super()
        this.added_notification = new Notification()
    }

}


